test/alternator/conftest.py

```diff
@@ -129,15 +128,15 @@ def dynamodb(request):
             region_name='us-east-1', aws_access_key_id=user, aws_secret_access_key=secret,
             config=boto_config.merge(botocore.client.Config(retries={"max_attempts": 0}, read_timeout=300)))
 
-def new_dynamodb_session(request, dynamodb):
+def new_dynamodb_session(request, dynamodb, user='cassandra', password='secret_pass'):
```

Comment:
This change is wrong... The "password" isn't used below (?) and the "user" should be called "role".